test/services/dumpers/agent-nodejs/agent-nodejs-dumper-mongoose-models.test.ts

```diff
@@ -47,114 +48,119 @@ describe('services > dumpers > agentNodejsDumper > mongoose models', () => {
         dbConfig: config.dbConfig,
         forestAuthSecret: 'forestAuthSecret',
         forestEnvSecret: 'forestEnvSecret',
+        language: Languages.Javascript,
       },
       schema,
     );
   }
 
-  describe('dump model files', () => {
-    describe('dumping schema', () => {
-      const testCases = [
-        {
-          name: 'should handle deep nested fields',
-          schema: deepNested,
-          file: {
-            model: 'persons',
-            expectedFilePath: `${__dirname}/expected/mongo-models/deep-nested.expected.js`,
+  const languages = [{ name: Languages.Javascript, extension: 'js' }];
+
+  describe.each(languages)('language: %s', ({ name: language, extension }) => {
```

Comment:
```suggestion
  describe.each(languages)('language: $name', ({ name: language, extension }) => {
```

I suggest this, because otherwise you get this:
<img width="510" alt="Screenshot 2023-03-27 at 16 08 58" src="https://user-images.githubusercontent.com/12433882/227964228-2ca3f7e3-318e-4675-ad9f-5b18ce9999e7.png">